### ISSUE TITLE
Program crashes often in Win 10, especially so if I click through too quickly

### ISSUE THREAD

**Issue by bradley1martin** [files-community/Files#5227]
And I LOVE the program, btw. Thanks, Brad

**Comment by gave92** [comment]
Thanks for this report. Could you add some more details of how to reproduce the crash?
Also please add the app's version to the description and upload the debug files.

**Comment by bradley1martin** [comment]
Hi. I just added the program recently, after reading about it in a syndicated article by the gal who writes a tech column.  I have version 1.5.0.0.  All I can add is that it seems if I click to go some files, I see what I want and when I quickly click on it, the program just goes "poof" and turns off.  So, I often go back to Windows Explorer.  
I really like your Preview mode.  Sometimes WE doesn't work at all.  Thanks, Brad

**Comment by gave92** [comment]
Could you try the following and see if the app is more stable?

Workaround 1:

1. Turn Off "Enable individual preferences for individual directories" in Settings > Files and Folders
2. Set Tiles or GridView layout mode

-------------------------------------

Workaround 2:

1. Disable these two settings (turn off caching) under Experimental

![image](https://user-images.githubusercontent.com/9673091/110851026-ce2f6680-82b0-11eb-8974-599e144ff399.png)

**Comment by shey2k7** [comment]
The app crashes when i open folders between 4GB and 7GB, with alot of files

**Comment by bradley1martin** [comment]
I changed the first Workaround item, but I can't find "Set Tiles or GridView layout mode" ?  

I disabled the middle two Experimental items, but realize now that I'd missed the other two.   I had a crash when opening a folder. An image is attached. 
I looked at this again, and I NOW have all Experimental items disabled, after that crash.  Hopefully it will be stable now. Thanks
![FILES](https://user-images.githubusercontent.com/85904877/122307677-ace9a200-cebf-11eb-820c-50014053f03a.JPG)
